opennlp-uima/src/test/java/opennlp/uima/dictionary/DictionaryResourceTest.java

```diff
@@ -74,7 +74,7 @@ public void testDictionaryWasLoaded() {
           .getResource("/opennlp.uima.Dictionary");
       // simple check if ordering always is the same...
```

Comment:
I'm wondering what this comment means. Looks like the developer was aware of the order issue and experimenting with two kinds of assertions but the test looks unfinished...

Reply:
The order was initially returned while using the HashSet.  I suppose the developer entered that returned order inside the assert statement.